controller/users_blackbox_test.go

```diff
@@ -90,6 +98,15 @@ func (s *TestUsersSuite) TestUpdateUserOK() {
 	assert.Equal(s.T(), newImageURL, *result.Data.Attributes.ImageURL)
 	assert.Equal(s.T(), newBio, *result.Data.Attributes.Bio)
 	assert.Equal(s.T(), newProfileURL, *result.Data.Attributes.URL)
+	updatedContextInformation := result.Data.Attributes.ContextInformation
```

Comment:
Thanks for pointing out - I ended up added a couple of tests.
1. PATCH operation without `contextInformation`. The service layer considers it `nil` at the payload level. So we are good.

tested out by doing a 

```
./bin/alm-cli update users --key $ALM_TOKEN  --payload '{ "data":{ "type":"identities" ,  "attributes":{"contextInformation":null,"bio":"something222"}}}'  -H localhost:8080 --pp
```
And I got a neat 200OK .

2. Added code / test to 'unset' stuff inside contextInformation 

```
$./bin/alm-cli update users --key $ALM_TOKEN  --payload '{ "data":{ "type":"identities" ,  "attributes":{"contextInformation":{"a":"a"},"bio":"something222"}}}'  -H localhost:8080 --pp
2017/03/17 15:11:09 [INFO] started id=v73ADIGK PATCH=http://localhost:8080/api/users
2017/03/17 15:11:09 [INFO] completed id=v73ADIGK status=200 time=13.368644ms
{
    "data": {
        "attributes": {
            "bio": "something222",
            "contextInformation": {
                "a": "a"
            },
            "email": "testuser2@domain.com",
            "fullName": "Test2 User2",
            "imageURL": "https://www.gravatar.com/avatar/47e397200125043ae3ef960944b2a5e1.jpg",
            "providerType": "kc",
            "url": "",
            "username": "testuser2"
        },
        "id": "71f343e3-2bfa-4ec6-86d4-79b91476acfc",
        "links": {
            "self": "http://localhost:8080/api/users/71f343e3-2bfa-4ec6-86d4-79b91476acfc"
        },
        "type": "identities"
    }
}

$ ./bin/alm-cli update users --key $ALM_TOKEN  --payload '{ "data":{ "type":"identities" ,  "attributes":{"cotextInformation":{"a":null},"bio":"something222"}}}'  -H localhost:8080 --pp
2017/03/17 15:11:18 [INFO] started id=pjF+davi PATCH=http://localhost:8080/api/users
2017/03/17 15:11:18 [INFO] completed id=pjF+davi status=200 time=16.582243ms
{
    "data": {
        "attributes": {
            "bio": "something222",
            "email": "testuser2@domain.com",
            "fullName": "Test2 User2",
            "imageURL": "https://www.gravatar.com/avatar/47e397200125043ae3ef960944b2a5e1.jpg",
            "providerType": "kc",
            "url": "",
            "username": "testuser2"
        },
        "id": "71f343e3-2bfa-4ec6-86d4-79b91476acfc",
        "links": {
            "self": "http://localhost:8080/api/users/71f343e3-2bfa-4ec6-86d4-79b91476acfc"
        },
        "type": "identities"
    }
}
```